.travis.yml

```diff
@@ -396,6 +396,19 @@ jobs:
       script:
         - travis_retry ./CocoapodsIntegrationTest/scripts/build_with_environment.sh --gemfile=./CocoapodsIntegrationTest/TestEnvironments/${POD_CONFIG_DIR}/Gemfile --podfile=./CocoapodsIntegrationTest/TestEnvironments/${POD_CONFIG_DIR}/Podfile
 
+    # FIS
+    - stage: test
+      env:
+        - PROJECT=Installations PLATFORM=iOS METHOD=pod-lib-lint
+      before_install:
+        - ./scripts/install_prereqs.sh
+      script:
+        - travis_retry ./scripts/pod_lib_lint.rb FirebaseInstallations.podspec --platforms=ios,tvos
```

Comment:
Please update and use if_changed.sh - https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/if_changed.sh